rllib/algorithms/ppo/torch/ppo_torch_learner.py

```diff
@@ -55,12 +57,8 @@ def possibly_masked_mean(data_):
         else:
             possibly_masked_mean = torch.mean
 
-        action_dist_class_train = (
-            self.module[module_id].unwrapped().get_train_action_dist_cls()
-        )
-        action_dist_class_exploration = (
-            self.module[module_id].unwrapped().get_exploration_action_dist_cls()
-        )
+        action_dist_class_train = module.get_train_action_dist_cls()
+        action_dist_class_exploration = module.get_exploration_action_dist_cls()
```

Comment:
Hey, I have a question here: shouldn't `exploration` **or** `inference` dist be used here? In a similar fashion to `GetActions` connector's [logic](https://github.com/sven1977/ray/blob/81da5de54b94c1c5d0287602ab0c753fdf99fa84/rllib/connectors/module_to_env/get_actions.py#L74-L77)?